Install issue

<!--
Welcome to the go-ipfs bug tracker. This is for you! Please read, and then delete this text before posting it.

If you haven't yet searched the issue tracker for an existing report concerning your issue, please do so now.

The go-ipfs issues are only for bug reports and directly actionable feature requests. Read https://github.com/ipfs/community/blob/master/contributing.md#reporting-issues if your issue doesn't fit either of those categories.

If you have a *SUPPORT QUESTION*, please direct it to our forum at https://discuss.ipfs.io.

Read https://github.com/ipfs/go-ipfs/blob/master/docs/github-issue-guide.md if you are not sure how to fill in this issue.
-->

#### Version information:
```
Software:

    System Software Overview:

      System Version: macOS 10.13.6 (17G65)
      Kernel Version: Darwin 17.7.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
```

 ```
$ go version
go version go1.11.1 darwin/amd64
```
<!--
Output From `ipfs version --all`

Please check dist.ipfs.io for a newer version of go-ipfs
and update if neccessary, then check again if the problem persists.
-->

#### Type:
<!-- Bug, Feature, Enhancement, Etc -->
Bug
#### Description:
<!--
This is where you get to tell us what went wrong or what feature you need. When doing so, please make sure to include *all* relevant information.

When requesting a feature, please be sure to include:

* Your motivation. Why do you need the feature?
* How the feature should work.

When reporting a bug, please try to include:

* What you were doing when you experienced the bug.
* Any error messages you saw, *where* you saw them, and what you believe may have caused them (if you have any ideas).
* When possible, steps to reliably produce the bug.
-->
Get the error, when try to install

```
make install
bin/check_go_version 1.11
fatal: 'go version' failed with output: go: parsing $GOFLAGS: non-flag "\"github.com/ipfs/go-ipfs\".CurrentCommit=3f7668bdc\""
make: *** [check_go_version] Error 1
```

executed from
```
 $pwd
/Users/theodor/go/src/github.com/ipfs/go-ipfs

 ~/go/src/github.com/ipfs/go-ipfs   master                                                                                                                                                                            ✔  1294  11:51:59
 $ echo $GOPATH
/Users/theodor/go
```

--- a/mk/golang.mk
+++ b/mk/golang.mk
@@ -5,6 +5,7 @@ GO_MIN_VERSION = 1.11
 # pre-definitions
 GOCC ?= go
 GOTAGS ?=
+unexport GOFLAGS
 GOFLAGS ?=
 GOTFLAGS ?=
 

